.cirrus.yml

```diff
@@ -1,12 +1,16 @@
 freebsd_instance:
   image_family: freebsd-13-0
+  cpu: 1
+  memory: 4G
 
 test_task:
   name: "Tests / FreeBSD / "
   only_if: $CIRRUS_TAG == ''
   skip: "!changesInclude('.cirrus.yml', 'poetry.lock', 'pyproject.toml', '**.json','**.py')"
   env:
     matrix:
+      - PYTHON: python3.6
```

Comment:
Note that with the scaled down VMs, all FreeBSD tests run in about the same time as the slowest Github Actions tests (4 minutes) since they're fully parallel.